gcloud/monitoring/client.py

```diff
@@ -195,6 +203,121 @@ def metric_descriptor(self, type_,
             display_name=display_name,
         )
 
+    @staticmethod
+    def metric(type_, labels):
+        """Construct a metric object.
+
+        The type of the Metric should match the type of a MetricDescriptor.
+        For a list of built-in Metric types and the associated labels, see:
+
+        https://cloud.google.com/monitoring/api/metrics
+
+        Here is an example of creating using a built-in metric type:
+
+             >>> metric = client.metric('instance/cpu/utilization' labels={
+             ...    'instance_name': 'my-instance',
+             ... })
+
+        For custom metrics, the simplest way to get the type is from the
+        :class:`~gcloud.monitoring.metric.MetricDescriptor` class used to
+        create the custom metric::
+
+             >>> metric = client.metric(metric_descriptor.type, labels={
+             ...    'instance_name': 'my-instance',
+             ... })
+
+        :type type_: string
+        :param type_: The metric type name.
+
+        :type labels: dict
+        :param labels: A mapping from label names to values for all labels
+                       enumerated in the associated :class:`MetricDescriptor`.
+
+        :rtype: :class:`~gcloud.monitoring.metric.Metric`
+        :returns: The Metric created with the passed in arguments
+        """
+        return Metric(type=type_, labels=labels)
+
+    @staticmethod
+    def resource(type_, labels):
+        """Construct a resource object.
+
+        For a list of possible MonitoredResources and their associated labels,
+        see:
+
+        https://cloud.google.com/monitoring/api/resources
+
+        For example, to create a Resource object for a GCE instance::
+
+            >>> type_ = 'https://cloud.google.com/monitoring/api/resources'
+            ... resource = client.resource(type_, labels= {
+            ...     'project_id': 'my-project-id',
+            ...     'instance_id': 'my-instance-id',
+            ...     'zone': 'us-central1-f'
+            ... })
+
+        :type type_: string
+        :param type_: The monitored resource type name.
+
+        :type labels: dict
+        :param labels: A mapping from label names to values for all labels
+            enumerated in the associated :class:`ResourceDescriptor`.
+
+        :rtype: :class:`gcloud.resource.Resource`
+        :returns: The Resource created with the passed in arguments
+        """
+        return Resource(type_, labels)
+
+    @staticmethod
+    def timeseries(value, metric, resource,
```

Comment:
Please move `value` after `metric` and `resource`, so that we will have the option of allowing the user to alternatively specify `point` or `points` later.